src/ui/layout/pane.vue

```diff
@@ -70,20 +71,55 @@ export default {
         this.type = this.$parent.type;
         this.styleProp = (this.type === 'horizontal') ? 'width' : 'height';
     },
+    async mounted() {
+        await this.$nextTick();
+        // Hide tree and/or inspector pane if specified in URL
+        this.handleHideUrl();
+    },
     methods: {
+
         toggleCollapse: function () {
             this.collapsed = !this.collapsed;
             if (this.collapsed) {
-                // Pane is expanded and is being collapsed
-                this.currentSize = (this.dragCollapse === true) ? this.initial : this.$el.style[this.styleProp];
-                this.$el.style[this.styleProp] = '';
+                this.handleCollapse();
             } else {
-                // Pane is collapsed and is being expanded
-                this.$el.style[this.styleProp] = this.currentSize;
-                delete this.currentSize;
-                delete this.dragCollapse;
+                this.handleExpand();
+            }
+        },
+        handleHideUrl: function () {
+            const HIDE_TREE_PARAM = 'hideTree';
```

Comment:
Let's move these consts outside of the method at the top of the script.

Reply:
Hi @shefalijoshi I've moved them.